src/screens/WalletConnect/Scan.js

```diff
@@ -0,0 +1,48 @@
+/* @flow */
+import React, { PureComponent } from "react";
+import Config from "react-native-config";
+import { Clipboard } from "react-native";
+import Scanner from "../../components/Scanner";
+import { ScreenName } from "../../const";
+import { TrackScreen } from "../../analytics";
+
+type Props = {
+  navigation: any,
+  route: { params: RouteParams },
+};
+
+type RouteParams = {
+  accountId: String,
+};
+
+type State = {};
+
+class ScanWalletConnect extends PureComponent<Props, State> {
+  componentDidMount() {
+    if (Config.MOCK_SCAN_WALLETCONNECT) {
+      setTimeout(async () => {
```

Comment:
this timeout should be cleared on Unmount in case user abort before.

Reply:
will change